<launch>
    <node name="pub_kpt_fbf_raw" pkg="sitl_dvrk" type="pub_kpt_fbf_raw.py" output="screen">
        <param name="calib_dir"           type="str"     value="L2R"/>
        <param name="model_path"          type="str"     value="/home/$(env USER)/dt2_dataset/kpts/fbf_results/model_final.pth"/>
        <param name="model_score_thr"     type="double"  value="0.1"/>
        <param name="window_size"         type="int"     value="20"/>
        <param name="mad_thr"             type="double"  value="2"/>
        <param name="kpt_score_thr"       type="double"  value="0.01"/>
    </node>
    <node name="pub_kpt_fbf_cp" pkg="sitl_dvrk" type="pub_kpt_fbf_cp.py" output="screen">
        <param name="tf_path"             type="str"     value="/home/$(env USER)/aruco_data/base_tfs.yaml"/>
        <!-- <param name="frame_id"            type="str"     value="ecm_left"/> -->
        <param name="frame_id"            type="str"     value="ecm_tip"/>
        <param name="tip_child_frame_id"  type="str"     value="dt2_psm2_tip"/>
        <param name="jaw_child_frame_id"  type="str"     value="dt2_psm2_jaw"/>
    </node>
    

    <?ignore
    
    ?>
</launch>